Makefile

```diff
@@ -85,3 +85,11 @@ docker-internal-linkcheck:
 container-internal-linkcheck: link-checker-image-pull
 	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --config config.toml,linkcheck-config.toml --buildFuture
 	$(CONTAINER_ENGINE) run --mount type=bind,source=$(CURDIR),target=/test --rm wjdp/htmltest htmltest
+
+clean-api-reference:
+	rm -rf content/en/docs/reference/kubernetes-api/ref/
+
+api-reference: clean-api-reference ## Build the API reference pages. go needed
+	mkdir content/en/docs/reference/kubernetes-api/ref/
```

Comment:
Why does the directory need to be created every time?

Reply:
Because the directory has been removed by the `clean-api-reference` dependence.